cli-config/src/lib.rs

```diff
@@ -1,3 +1,55 @@
+//! Loading and saving the Solana CLI configuration file.
+//!
+//! The configuration file used by the Solana CLI includes information about the
+//! RPC node to connect to, the path to the user's keypair, and more. Other
```

Comment:
```suggestion
//! RPC node to connect to, the path to the user's signer, and more. Other
```

Reply:
Same question about my other keypair-to-signer suggestion

Reply:
Yeah I'll rework it to make it more correct.